pkg/e2e/e2e-api/http.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2022 The Numaproj Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"bytes"
+	"crypto/tls"
+	"encoding/json"
+	"fmt"
+	"github.com/numaproj-contrib/numaflow-utils-go/testing/fixtures"
+
+	"io"
+	"log"
+	"net/http"
+)
+
+var httpClient *http.Client
+
+func init() {
+	httpClient = &http.Client{
+		Transport: &http.Transport{
+			TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
+		},
+	}
+
+	// send-message API is used to post data to a http source vertex pod.
+	// The API takes in two parameters(podIp and vertexName) and constructs the target url as
+	// https://{podIp}:8443/vertices/{vertexName}.
+	http.HandleFunc("/http/send-message", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Do we use http to send messages for sqs source e2e test? I think we can remove some code under the `e2e-api` folder. e.g. nats and kafka is not used. Please determine what is used by sqs source e2e and remove those not used.

Reply:
yes we use http

Reply:
Sorry, I meant that we don't use `/http/send-message` api to send messages. Instead we use sqs source. We can remove this particular handle Func.